### PR TITLE
fix(proxy): simplify URL construction in pagination links (#36)

### LINKED ISSUE #36: Issue with proxy forwarder
### Problem
I deployed the website on a container, and experienced a problem when clicking on the pagination.
I have been redirected to `http://localhost:3000/?query=&sort=yotas_desc&page=1`, who is not normal.

### Expected behavior
Be redirected to `https://potential-meme-xxxxxxx-3000.app.github.dev/?query=&sort=yotas_desc&page=1`

### Test environment
GitHub codespace.

### Possible solution
- remove full absolute URL build in `src/views/partials/pagination.hbs`.

### PR DESCRIPTION
**Pull request by sikatikenmogne**
## Related Issues
- Fixes #36 

## Changes (initial)
- Added a new helper function called `removeDomain` to handle URL parsing and remove the domain from the given full URL.
- Updated the `constructUrl` helper function to use `removeDomain` for constructing clean URLs.
- Modified the `pagination.hbs` template to use `removeDomain` in the `href` attribute of the page links.

### After review
- Simplifying URL construction by removing @root.fullUrl host part

## Preview
https://github.com/user-attachments/assets/00a01939-5647-48f0-b759-0f4f70cb1e13

### Tested environments
- Github Codespaces
- Windows
- Microsoft DevContainer

### CODE REVIEW

File: src/helpers/handlebars.ts [hunk 1]
```diff
@@ -47,4 +48,13 @@ export const repoName = (v1: string): string => {
 
 export const addClassIfEqual = (v1: number, v2: number, extra: number, v3: string): string => {
   return v1 === (v2+extra) ? v3 : '';
+};
+
+export const removeDomain = (fullUrl: string): string => {
```
Comment (pythonbrad) [inline src/helpers/handlebars.ts:53]: I don't think that we need to add a helper for it.

This `fullUrl` is declared at [/src/index.ts#L66](https://github.com/osscameroon/miniyotas/blob/7610ee6837aac63d76f4483f48891a8cab528be5/src/index.ts#L66), and looks used only in this `pagination.hbs` file.

**Comment by sikatikenmogne** [comment]
Hi @pythonbrad, thanks' for your review. I've applied your suggestion and reverted my commit about the helper function.
If you decide not to merge this branch, I’ve also opened a separate PR (#38) that addresses only the suggested changes. I hope you will appreciate it.

**Comment by pythonbrad** [comment]
Ok good, when I will test it, I will send you a feedback.

**Comment by pythonbrad** [comment]
@sikatikenmogne, it's not good to push more than one pr who solve the same issue.

**Comment by sikatikenmogne** [comment]
> @sikatikenmogne, it's not good to push more than one pr who solve the same issue.

I know, I just thought it would be better to summarize the changes in this conversation in a one commit pull request instead of a 3 which looks a bit messy, feel free to close the request sweater that seems like too much.

**Comment by sikatikenmogne** [comment]
I've closed the PR #38

**Comment by sikatikenmogne** [comment]
@rakici

**Review by rakici — APPROVED** [review]
Nice catch.

**Pull request merged**